src/openrct2-ui/windows/TitleScenarioSelect.cpp

```diff
@@ -226,10 +226,17 @@ static void window_scenarioselect_init_tabs(rct_window* w)
         }
     }
 
-    int32_t firstPage = bitscanforward(showPages);
-    if (firstPage != -1)
+    if (showPages & 1 << gConfigInterface.scenarioselect_last_tab)
```

Comment:
For clarity, I would prefer you change this to:
```suggestion
    if (showPages & (1 << gConfigInterface.scenarioselect_last_tab))
```